login.css

```diff
@@ -109,7 +109,7 @@ h3{
 p{
 	margin-left:130px;
 	margin-top:250px;
-	font-size:50px;
+	font-size:24px;
```

Comment:
04/08/2021: After trialing the font-size I released it the font size for heading was too big So I make an update for it to 24 px which I think it is very reliable to read for all people(old and young people)